### PR TITLE
Update reading/writing without interruption after pod restart e2e test

### PR DESCRIPTION
**Pull request by torredil**
**What is this PR about? / Why do we need it?**

This PR has all the necessary components to enable running kOps / EKS e2e CI jobs in the latest version of K8s without `hostNetwork: true`. Will rebase after #1087 and #1054.

**What testing is done?** 

CI

### CODE REVIEW

File: hack/kops-patch.yaml [hunk 1]
```diff
@@ -8,8 +8,14 @@ spec:
...
+            "elasticfilesystem:DescribeAccessPoints",
+            "elasticfilesystem:DescribeMountTargets",
+            "ec2:DescribeAvailabilityZones"
           ],
           "Resource": "*"
         }
       ]
+
+  cloudConfig:
+    awsEBSCSIDriver:
```
Comment (Ashley-wenyizha) [inline hack/kops-patch.yaml:20]: `awsEFSCSIDriver`?

Comment (mjsoyeon) [reply hack/kops-patch.yaml:20]: When k8s is installed through kops, ebs driver enables as a default since k8s 1.22

Comment (Ashley-wenyizha) [reply hack/kops-patch.yaml:20]: when you say `This config managed ebs driver`, could you forward the link of the resource? and what do you mean by `ebs driver enables as a default`?

Comment (mjsoyeon) [reply hack/kops-patch.yaml:20]: [New clusters running Kubernetes 1.22 will have AWS EBS CSI driver enabled by default.
](https://github.com/kubernetes/kops/blob/master/docs/releases/1.22-NOTES.md#:~:text=New%20clusters%20running%20Kubernetes%201.22%20will%20have%20AWS%20EBS%20CSI%20driver%20enabled%20by%20default.)

Comment (Ashley-wenyizha) [reply hack/kops-patch.yaml:20]: I guess what I am confused here is I do not fully understand how is this controlled and why do we need to control it through ebs driver?

Could you add more details/explanations here? thanks

Comment (torredil) [reply hack/kops-patch.yaml:20]: @Ashley-wenyizha To clarify, this patch simply disables the installation of the EBS CSI Driver during cluster creation with kOps. This is not being through the EBS CSI Driver -- it is part of kOps cluster configuration.

Comment (Ashley-wenyizha) [reply hack/kops-patch.yaml:20]: yep I have communicated offline with Soyeon. lgtm

**Pull request merged**